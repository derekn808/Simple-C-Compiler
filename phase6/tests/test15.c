int main(void)
{
    double a, b, c;

    a = 1.1e1;
    b = 2.213;
    c = 3.321;

    printf("%g\n", a + b);
    printf("%g\n", a * b);
    printf("%g\n", c - b);

    c = c / b + a * 3.14159;

    printf("%d\n", a > b);
    printf("%d\n", a == b);
    printf("%d\n", c <= b);

    printf("%d\n", a < b);
    printf("%d\n", a != b);
    printf("%d\n", c >= b);
}
