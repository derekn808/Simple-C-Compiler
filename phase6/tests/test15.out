13.213
24.343
1.108
1
0
0
0
1
1
